common/searchattribute/validator.go

```diff
@@ -68,69 +68,124 @@ func NewValidator(
 	}
 }
 
+func (v *Validator) getAlias(saFieldName string, namespace string) (string, error) {
+	var err error
+	saAlias := saFieldName
+	if IsMappable(saFieldName) && v.searchAttributesMapper != nil {
+		saAlias, err = v.searchAttributesMapper.GetAlias(saFieldName, namespace)
+	}
+	return saAlias, err
```

Comment:
```suggestion
	return saFieldName, nil
```
and you don't need to vars above.